src/screens/ProductSummary/index.tsx

```diff
@@ -48,64 +49,21 @@ const ProductSummary = () => {
             data,
             (item: { quantityOnHand: number }) => item.quantityOnHand > 0
           );
+          state.productData = state.productSummary
```

Comment:
What's the idea behind this? Is there a need to have both `productData` and `productSummary`?

Reply:
Make a copy of the original to reset back to load all values